test/Tmds.Ssh.Tests/RemoteProcessTests.cs

```diff
@@ -44,6 +44,36 @@ public async Task HelloWorld(bool stderr)
         Assert.Equal(0, process.ExitCode);
     }
 
+    [Fact]
+    public async Task SubsystemProcess()
+    {
+        using var client = await _sshServer.CreateClientAsync();
+        byte[] helloWorld1Bytes = Encoding.UTF8.GetBytes("hello world 1");
+        byte[] helloWorld2Bytes = Encoding.UTF8.GetBytes("hello world 2");
+        using var process = await client.ExecuteSubsystemAsync(_sshServer.TestSubsystem);
+
+        await process.WriteLineAsync("echo -n 'hello world 1'");
+
+        byte[] buffer = new byte[512];
+        (bool isError, int bytesRead) = await process.ReadAsync(buffer, buffer);
+        Assert.False(isError);
+        Assert.Equal(helloWorld1Bytes, buffer.AsSpan(0, bytesRead).ToArray());
+
+        await process.WriteLineAsync("echo -n 'hello world 2'");
```

Comment:
I tried to do `>&2` to test out `isError` but it seems like SSH doesn't return `SSH_EXTENDED_DATA_STDERR` on subsystem based channels. I was unable to even get the `ssh` binary to return stderr on a pre-configured subsystem that wrote to stderr so I assume it's not something that can happen. I couldn't find anything in the RFC around this but I didn't look too hard.

Reply:
The test is good as is. It covers what we need.